stanford_events.module

```diff
@@ -171,11 +171,23 @@ function stanford_events_node_view(array &$build, EntityInterface $entity, Entit
  *   The date formatted with stanford_long_day_date_year
  */
 function stanford_events_schedule_format_date($raw, $obj) {
```

Comment:
Not a change in this PR, but lets change the `$obj` to `ContentEntityInterface $entity`. This will make it much easier to understand whats happening.

Reply:
I just went ahead and changed it here; might as well take care of it now.